includes/class-wc-taxjar-integration.php

```diff
@@ -638,8 +638,14 @@ public function calculate_tax( $options = array() ) {
 			$taxes['freight_taxable']    = (int) $taxjar_response->freight_taxable;
 			$taxes['has_nexus']          = (int) $taxjar_response->has_nexus;
 			$taxes['tax_rate']           = $taxjar_response->rate;
```

Comment:
@dallendalton Do we still use `$taxes['tax_rate']` anywhere after this change?

Reply:
Nope we don't, I have removed it. Thanks!